README.md

```diff
@@ -1,7 +1,109 @@
 # Pipeline Stage View Plugin
 
-This plugin provides a graphical swimlane view of pipeline stage execution (as well as a build history of the stages). 
+# Stage View: 
+
+## ![](docs/images/who-broke-it.png)
+
+## ![](docs/images/green-and-mean.png)
+
+When you have complex builds Pipelines, it is useful to be able to see
+the progress of each stage.  The Pipeline Stage View plugin includes an
+extended visualization of Pipeline build history on the index page of a
+flow project, under *Stage View*. (You can also click on *Full Stage
+View* to get a full-screen view.)
+
+To take advantage of this view, you need to define stages in your flow.
+You can have as many stages as you like, in a linear sequence. (They may
+be inside other steps such as 'node' if that is convenient.) Give each a
+short name that will appear in the GUI.
+
+``` syntaxhighlighter-pre
+node {
+  stage ('Checkout') {
+    svn 'https://svn.mycorp/trunk/'
+    stage 'Build'
+    sh 'make all'
+    stage 'Test'
+    sh 'make test'
+  }
+}
+```
+
+When you run some builds, the stage view will appear
+with *Checkout*, *Build*, and *Test* columns, and one row per build.
+When hovering over a stage cell you can click the *Logs* button to see
+log messages printed in that stage: after the preceding stage step and
+before the next one.
+
+Other important events are also indicated with special buttons, either
+in a stage cell or for the build as a whole. If a build is waiting in
+an input step, you will be able to tell it to proceed (or
+not). *Download* can be used to obtain any artifacts archived by a
+build. 
+
+Each row also records the build number, date it was started, and any
+changelog entries from a version control system. Progress bars also
+indicate how long each stage is taking and how long it might still be
+expected to take, based on historical averages.
+
+# Usage Notes/Limits:
+
+-   Block-scoped stages (stages that take a closure, see below) may
+    occur in parallels but are not displayed in parallel.
+
+    ``` syntaxhighlighter-pre
+    stage('name'){ echo 'steps here' }
+    ```
+
+&nbsp;
+
```

Comment:
Generates more whitespace than we probably need. As rendered in Chrome:

![image](https://user-images.githubusercontent.com/21689198/70249477-6b186100-174a-11ea-8e12-9fed15f7b4ec.png)